docs/sources/operations/shuffle-sharding.md

```diff
@@ -0,0 +1,109 @@
+
+## The issues that shuffle sharding mitigates
+
+Shuffle sharding can be configured for the query path.
+
+The query path is sharded by default, and the default does not use shuffle sharding.
+Each tenant’s query is sharded across all queriers, so the workload uses all querier instances.
+
+In a multi-tenant cluster, sharding across all instances of a component may exhibit these issues:
+
+- Any outage of a component instance affects all tenants
+- A misbehaving tenant affects all other tenants
+
+An individual query may create issues for all tenants.
+A single tenant or a group of tenants may issue an expensive query:
+one that causes a querier component to hit an out-of-memory error,
+or one that causes a querier component to crash.
+Once the error occurs,
+the tenant or tenants issuing the error-causing query will be reassigned
+to other running queriers,
+up to the limit imposed by the `max_queriers_per_tenant` configuration.
+This, in turn, may affect the queriers that have been reassigned.
+
+## How shuffle sharding works
+
+The idea of shuffle sharding is to assign each tenant to a shard composed by a subset of the Loki queriers, aiming to minimize the overlapping instances between distinct tenants.
+
+A misbehaving tenant will affect only its shard's queriers. Due to the low overlap of queriers among tenants, only a small subset of tenants will be affected bythe misbehaving tenant.
+Shuffle sharding requires no more resources than the default sharding strategy.
+
+Shuffle sharding does not fix all issues.
+If a tenant repeatedly sends a problematic query, the crashed querier
+will be disconnected from the query-frontend, and a new querier
+will be immediately assigned to the tenant’s shard.
+This invalidates the positive effects of shuffle sharding.
+In this case,
+configuring a delay between when a querier disconnects because of a crash,
+and when the crashed querier is actually removed from the tenant’s shard
+and another healthy querier is added as a replacement improves the situation.
+A delay of 1 minute may be a reasonable value in
+the query-frontend with configuration parameter
+`-query-frontend.querier-forget-delay=1m`, and in the query-scheduler with configuration parameter
+`-query-scheduler.querier-forget-delay=1m`.
+
+### Low probability of overlapping instances
+
+If an example Loki cluster runs 50 queriers and assigns each tenant 4 out of 50 queriers, shuffling instances between each tenant, there are 230K possible combinations.
+
+Statistically, randomly picking two distinct tenants, there is:
+
+- a 71% chance that they will not share any instance
+- a 26% chance that they will share only 1 instance
+- a 2.7% chance that they will share 2 instances
+- a 0.08% chance that they will share 3 instances
+- only a 0.0004% chance that their instances will fully overlap
+
+![overlapping instances probability](./shuffle-sharding-probability.png)
```

Comment:
I imagine that you're looking at the representation of this section in Github.  We need the relative path to the image to build in Hugo, for hosting our docs.  That extra period is needed to do that.  Can you please add it back in?  Then I'll approve this PR and merge it in!
```suggestion
![overlapping instances probability](../shuffle-sharding-probability.png)
```

Reply:
Added your change. But it's confusing for any doc's author and hard to verify if image rendering works fine.